kwok/entrypoint.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+kwokctl start cluster "$@"
```

Comment:
in the original dockerfile I can see that entrypoint scrip also brings up a proxy to listen on  the API port and forward requests to the cluster. Are you sure we don't need that? Did you try connecting to this image from outside?

Reply:
Since you have mentioned removing it from the entry point I have removed it

Reply:
I mentioned removing the part that prints out the connection details, not the part that brings up the proxy. If you are able to connect to the cluster without it then by all means, lets keep things simple. By the way how to we set the port for the API server - we need to be able to customize that at image load time to avoid colliding with other ports on the user's machine.

Reply:
Done